src/layout/msWorkspace/msWindow.ts

```diff
@@ -29,6 +29,8 @@ import { PrimaryBorderEffect } from './tilingLayouts/baseResizeableTiling';
 
 const isWayland = GLib.getenv('XDG_SESSION_TYPE').toLowerCase() === 'wayland';
 
+const PLACEHOLDER_ICON_SIZE = 248;
```

Comment:
Very reasonable. If I'd suggest anything it would be `PLACEHOLDER_APP_ICON_SIZE`, but that's nitpicking.